charts/nes-node-web/values.yaml

```diff
@@ -8,6 +8,8 @@ port: ""
 host: ""
 version: ""
 urlPrefix: ""
+serviceSubdomain: ""
+clusterZone: ""
```

Comment:
Let's comment this out since it's required but should be provided at runtime by the ecosystem. That way we'll still see it as a thing, but it'll fail if you don't set it